src/server/.tests/routes/controllers/utilities/passportLoginCllbacks.spec.js

```diff
@@ -0,0 +1,156 @@
+const passportLoginCallbacks = require('../../../../routes/controllers/utilities/passportLoginCallbacks')
+const responseMessages = require('../../../../routes/controllers/utilities/responseMessages')
+const statusCodes = require('../../../../routes/controllers/utilities/statusCodes')
+const EventEmitter = require('events').EventEmitter
+const httpMocks = require('node-mocks-http')
+const assert = require('chai').assert
+let suite = {}
+
+describe('Passport Login Callbacks', () => {
+	beforeEach(() => {
+		suite.responseMock = httpMocks.createResponse({
+			eventEmitter: EventEmitter
+		})
+	})
+
+
+	describe('authenticate callback', () => {
+		beforeEach(() => {
+			suite.authenticateCallback = passportLoginCallbacks.authenticateCallback(null, suite.responseMock)
```

Comment:
Authenticate callback is a thing returned from authenticate callback?
When you take a flour, and bake it, do you have bread, or still a flour?

Reply:
ok